### ISSUE TITLE
Bug: getImageURI returns image with chart curve filled in dark grey in Firefox

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#1513]
```
What steps will reproduce the problem? Please provide a link to a
demonstration page if at all possible, or attach code.
1. create a LineChart and call the #getImageURI method.  Example: http://jsfiddle.net/asgallant/bCtx4/


What component is this issue related to (PieChart, LineChart, DataTable,
Query, etc)?
LineChart

Are you using the test environment (version 1.1)?
(If you are not sure, answer NO)
no

What operating system and browser are you using?
Firefox, Windows 7 x64


*********************************************************
For developers viewing this issue: please click the 'star' icon to be
notified of future changes, and to let us know how many of you are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `drew_gallant@abtassoc.com` on 2014-03-03 18:17:47

**Comment by orwant** [comment]
```
This should be fixed in the 1.1 development release as of late yesterday.

Please take a look.
```

Original issue reported on code.google.com by `jfaller@google.com` on 2014-03-05 15:04:15

**Comment by orwant** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `jfaller@google.com` on 2014-03-05 15:07:16
